entries/r/roborock.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Roborock": {
+    "domain": "roborock.com",
```

https://roborock.com/ doesn't work.
```suggestion
    "domain": "roborock.com",
    "url": "https://www.roborock.com/",
```

---

Maybe i'll change it to their global URL (https://global.roborock.com/) instead, the www one goes to their forums 🫤

---

Really? To me https://www.roborock.com/ redirects to https://us.roborock.com/ :laughing:

---

Weird, the 3 times I tried that link it was going to https://forum.roborock.com/ now it is going to the US site 🤷‍♂️